components/more-less/more-less.js

```diff
@@ -8,15 +8,41 @@ import { LocalizeCoreElement } from '../../lang/localize-core-element.js';
 import ResizeObserver from 'resize-observer-polyfill/dist/ResizeObserver.es.js';
 import { styleMap } from 'lit-html/directives/style-map.js';
 
+/**
+ * The `d2l-more-less` element can be used to minimize the display of long content, while providing a way to reveal the full content.
+ * @slot - Default content placed inside of the component
+ * @fires d2l-more-less-render - Dispatched when the component finishes rendering
+ */
 class MoreLess extends LocalizeCoreElement(LitElement) {
 
 	static get properties() {
 		return {
-			blurColor: { type: String, attribute: 'blur-color' }, // The gradient color of the blurring effect. Must be hex color code.
-			expanded: { type: Boolean, reflect: true }, // Indicates whether element is in "more" state.
-			hAlign: { type: String, attribute: 'h-align' }, // The h-align property of the more-less button.
-			height: { type: String }, // The maximum height of the content when in "less" state.
-			inactive: { type: Boolean, reflect: true }, // Whether the component is active or inactive.
+			/**
+			 * The gradient color of the blurring effect.
+			 * @type {"hex color code"}
```

Comment:
Probably want to use `string` here so that it's valid JSDoc, but maybe we can use the [typedef](https://jsdoc.app/tags-typedef.html) to define a custom HEX-string?

Reply:
So declaring it as a typedef makes sense (possibly more for the h-align case). It doesn't seem to affect the `custom-elements.json` file other than if I use `@type {HexString}` then that's noted as the type, so I don't know if that will potentially cause confusion. Where would be the best place to put the `@typedef`? Does right above `@type` make sense? I initially put them at the top of the file but that looks a bit cluttered.

Reply:
I'm not sure... I doubt there'd be a way to put it somewhere centralized such that we could share it across all the things that need HEX strings. We could also just go with `{string}` for now and defer figuring out complex types until later. :)